tox.ini

```diff
@@ -15,6 +15,14 @@ envlist =
 #     tox -l -v
 #
 
+[testenv:check-types]
+description = check type hints, e.g. with mypy
+deps =
+    mypy
+commands =
+    mypy --install-types --non-interactive
```

Comment:
```suggestion
```

Reply:
It looks like `--install-types` requires a previous run so let's try just installing the requests type stubs: https://github.com/spacetelescope/stcal/pull/267/files#r1684549714

Reply:
looks to be working, thanks!